src/node/state.ml

```diff
@@ -54,6 +55,10 @@ let make ~identity ~trusted_validator_membership_change
     { hash; data } in
   let initial_snapshots =
     Snapshots.make ~initial_snapshot ~initial_block ~initial_signatures in
+  let init_peers = [] in
+  (* FIXME: get peers *)
+  let pollinate_address = Pollinate.Address.create "0.0.0.0" 4242 in
```

Comment:
Can pipe if you'd like,
```ocaml
let pollinate_node =
  Pollinate.Address.create "0.0.0.0" 4242
  |> Pollinate.PNode.init ~init_peers
```
I tend to avoid unnecessary bindings if I can, but I'm not sure if there's any actual difference.

Reply:
Still working on it